Sources/ArcGISToolkit/Components/FeatureFormView/NavigationPathItem.swift

```diff
@@ -46,16 +46,15 @@ extension FeatureFormView {
                       .utilityAssociationFeatureSourcesView(_, elementB, filterB)):
                 elementA === elementB
                 && filterA === filterB
-            case let (.utilityAssociationFilterResultView(_, _, elementA, titleA),
-                      .utilityAssociationFilterResultView(_, _, elementB, titleB)):
+            case let (.utilityAssociationFilterResultView(_, _, elementA, filterA),
+                      .utilityAssociationFilterResultView(_, _, elementB, filterB)):
                 elementA === elementB
-                && titleA == titleB
-            case let (.utilityAssociationGroupResultView(_, _, elementA, filterTitleA, groupTitleA),
-                      .utilityAssociationGroupResultView(_, _, elementB, filterTitleB, groupTitleB)):
-                // TODO: Improve group identification (Apollo 1391).
+                && filterA == filterB
```

Comment:
Will chat with David. Currently seeing a build error on this line.

Reply:
Looks like I messed some lines up when I was resolving merge conflicts. Should be all fixed now.